CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ All notable changes to this extension will be documented in this file.
 ### Fixed
 
 - Improved resolving newly created direct connection connectivity state.
+- Fixed an issue where the extension would fail to sync CCloud authentication related user settings
```

Comment:
Will be separated from existing `main` fixes for the `1.0.2` release + changelog update.